NickvisionMoney.GNOME/Program.cs

```diff
@@ -95,7 +95,7 @@ public int Run(string[] args)
     /// </summary>
     /// <param name="sedner">Gio.Application</param>
     /// <param name="e">EventArgs</param>
-    private async void OnActivate(Gio.Application sedner, EventArgs e)
+    private void OnActivate(Gio.Application sedner, EventArgs e)
```

Comment:
```suggestion
    /// <param name="sender">Gio.Application</param>
    /// <param name="e">EventArgs</param>
    private void OnActivate(Gio.Application sender, EventArgs e)
 ```

Reply:
Oh I commented after merge :sweat_smile:

Reply:
thanks anyway, will be fixed in some other PR later :)

Reply:
Haha no worries :) fixed https://github.com/nlogozzo/NickvisionMoney/commit/7eeaffa7ff26d39f2cdaef3b24bf35565826bdd5